base/BUILD.gn

```diff
@@ -1005,6 +1005,12 @@ component("base") {
     ]
   }
 
+  if (is_cobalt && is_starboard) {
```

Comment:
Minimize the number of if statements by combining this one with the one below (and moving it to the bottom of the target with the rest of the general conditions)

https://gn.googlesource.com/gn/+/main/docs/style_guide.md#conditions